test/test.js

```diff
@@ -94,4 +95,41 @@ describe('absolution', function() {
     });
     assert.equal(result, expected);
   });
+
+  it('should handle the scrset in img', function() {
+    const result = absolution(`<img src="cat.jpg" alt="cat" 
```

Comment:
I would like to see this test match the test in the issue. It is typically a good pattern to demonstrate that the original problem raised in the issue has been fixed by your code change.

Reply:
I didn't understand what I need to do?